migration-engine/connectors/sql-migration-connector/src/sql_renderer/postgres_renderer.rs

```diff
@@ -777,7 +784,7 @@ fn render_alter_column(
 
                 after_statements.push(format!(
                     "ALTER SEQUENCE {sequence_name} OWNED BY {table_name}.{column_name}",
-                    sequence_name = Quoted::postgres_ident(sequence_name),
+                    sequence_name = sequence_name,
```

Comment:
tip: this line could go away, if `sequence_name` is not in the name parameters to `format!()`, it will fall back to the binding of the same name in the local scope